src/main/java/com/elmc/booking/adapters/incoming/rest/ErrorMessage.java

```diff
@@ -0,0 +1,11 @@
+package com.elmc.booking.adapters.incoming.rest;
+
+import lombok.Value;
+
+import java.time.LocalDateTime;
+
+@Value
```

Comment:
Skoro to Java 17 to czy nie fajniej byłoby użyć recordów?

Reply:
Rekordy nie mogą mieć dodatkowych pól poza tymi z konstruktora. ErrorMessage był początkowo rekordem ale podawanie za każdym razem daty przy tworzeniu obiektu jakoś średnio mi wyglądało, więc zmieniłem to na klasę z ustawioną datą.